### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by gilbertoalbino** [brahmosjs/brahmos#124]
It seems like this project is dead. Is it?

**Comment by OneHoopyFrood** [comment]
Same question. Has this been abandoned?

**Comment by Dev-Siri** [comment]
It's been more than 1 year since the last commit.
This project maybe dead at this point.

**Comment by s-yadav** [comment]
Yes. I decided to stop this project due to some other priorities. Definitely it had some good progress. But sadly I had to stop on this.

**Comment by gilbertoalbino** [comment]
Ok, I do understand. Thank you so far!

**Comment by Jbonez87** [comment]
Would you be open to allowing others to maintain this project?

**Comment by uniho** [comment]
Trying it out
https://github.com/uniho/potate

**Comment by s-yadav** [comment]
Nice @uniho. All the best on it. Also, create a sandbox or demo for it. Would love to try it out.
Btw, great name Potate 🙂

**Comment by uniho** [comment]
> Nice [@uniho](https://github.com/uniho). All the best on it. Also, create a sandbox or demo for it. Would love to try it out. Btw, great name Potate 🙂

@s-yadav 
Thanks! Your code is great.
Feel free to drop some advice or a PR anytime.

**Comment by uniho** [comment]
> Nice [@uniho](https://github.com/uniho). All the best on it. Also, create a sandbox or demo for it. Would love to try it out. Btw, great name Potate 🙂

Done🚀
https://stackblitz.com/edit/github-cywxrcwm-3n79zz3t?file=src%2Fcomponents%2FPotate.jsx
